packages/core/src/app/order/OrderSummarySubtotals.test.tsx

```diff
@@ -23,17 +25,19 @@ describe('OrderSummarySubtotals', () => {
         const order = getOrder();
 
         render(
-            <LocaleContext.Provider value={contextValue}>
-                <OrderSummarySubtotals
-                    coupons={order.coupons}
-                    discountAmount={order.discountAmount}
-                    giftCertificates={order.payments && mapFromPayments(order.payments)}
-                    handlingAmount={order.handlingCostTotal}
-                    shippingAmount={order.shippingCostTotal}
-                    subtotalAmount={order.orderAmount}
-                    taxes={order.taxes}
-                />
-            </LocaleContext.Provider>
+            <CheckoutProvider checkoutService={createCheckoutService()}>
```

Comment:
We have introduced a custom render to wrap all contexts.
`import { fireEvent, render, screen } from '@bigcommerce/checkout/test-utils';`
Try it out to avoid making context on your own.

Reply:
Thanks @bc-peng, thanks for this solution. I changed it for my test which uses react testing library